src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapAllSimulation.scala

```diff
@@ -19,7 +19,7 @@ class JmapAllSimulation extends Simulation {
 
   private val scenario = new JmapAllScenario()
 
-  setUp(scenario.generate(UserFeeder.toFeeder(users), Configuration.ScenarioDuration, RecipientFeeder.usersToFeeder(users))
+  setUp(scenario.generate(UserFeeder.toFeeder(users), Configuration.ScenarioDuration, RecipientFeeder.usersToFeeder(users).random)
```

Comment:
is there any case where we want a finite feeder? If not, we could put the random into usersToFeeder